schedulers-tests/src/test/java/org/mule/test/integration/SchedulerTestCase.java

```diff
@@ -54,6 +55,13 @@ protected String getConfigFile() {
     return "org/mule/test/integration/scheduler-config.xml";
   }
 
+  @Test
+  public void test() {
+    SchedulerMessageSource source = (SchedulerMessageSource) locator.find(buildFromStringRepresentation("scheduler")).get(0);
+    FixedFrequencyScheduler conf = (FixedFrequencyScheduler) source.getConfiguration();
+    System.out.println(conf.getFrequency());
```

Comment:
assert instead of sysout

Reply:
this should actually be deleted. This was just a doodling accident